testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/ApplicationsTest.java

```diff
@@ -32,4 +52,73 @@
     protected AbstractLoggedInPage getAccountPage() {
         return applicationsPage;
     }
+
+    @Override
+    public void addTestRealms(List<RealmRepresentation> testRealms) {
+        super.addTestRealms(testRealms);
+        RealmRepresentation realm = testRealms.get(0);
+
+        realm.setClients(Arrays.asList(
+                ClientBuilder
+                        .create()
+                        .clientId("always-display-client")
+                        .id(KeycloakModelUtils.generateId())
+                        .name("Always Display Client")
+                        .baseUrl(APP_ROOT + "/always-display-client")
+                        .directAccessGrants()
+                        .secret("secret1")
+                        .alwaysDisplayInConsole(true)
+                        .build(),
+                ClientBuilder
+                        .create()
+                        .clientId("third-party-client")
+                        .id(KeycloakModelUtils.generateId())
+                        .name("Third Party Client")
+                        .baseUrl(APP_ROOT + "/third-party-client")
+                        .directAccessGrants()
+                        .secret("secret1")
+                        .consentRequired(true)
+                        .build()
+        ));
+    }
+
+    @Test
+    public void applicationListTest() throws Exception {
+        List<ApplicationsPage.ClientRepresentation> applications = applicationsPage.getApplications();
+        assertFalse(applications.isEmpty());
+        Map<String, ApplicationsPage.ClientRepresentation> apps = applications.stream().collect(Collectors.toMap(x -> x.getClientId(), x -> x));
+        assertThat(apps.keySet(), containsInAnyOrder("always-display-client", "account-console"));
+
+        assertClientRep(apps.get("account-console"), "Account Console", false, true, "/realms/test/account/", false);
+        assertClientRep(apps.get("always-display-client"), "Always Display Client", false, false, "https://localhost:8543/auth/realms/master/app/always-display-client", false);
+    }
+
+    @Test
+    public void toggleApplicationDetailsTest() throws Exception {
+        List<ApplicationsPage.ClientRepresentation> applications = applicationsPage.toggleApplicationsDetails();
```

Comment:
We might want to try toggling only one app. What if the button is broken and toggles all apps at once instead of only one? :)